debian/changelog

```diff
@@ -0,0 +1,12 @@
+eos-data-distribution (0.1.0) UNRELEASED; urgency=medium
```

Comment:
hmm, I think that’s a bit pointless when there are realistically no other consumers of a project (it’s closed source, even), but whatever. I’ll split this into a separate branch when merging.